_pages/infrastructure/monitoring.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Monitoring
+parent: Infrastructure
+---
+
+There are a couple kinds of monitoring that you will need to have in place for any application: [errors](#errors) and [downtime](#downtime).
+
+Monitoring is only useful if the relevant people are alerted when something goes wrong. It might take some tweaking of the thresholds to get the signal-to-noise ratio right.
```

Comment:
Can we add some extra clauses to that first sentence? I've found it helpful to think about alerting in this way:

"Monitoring is only useful if:

- the relevant people are alerted when something goes wrong
- they consider these alerts worth investigating
- they have sufficient access and understanding to at least triage and escalate an alert, if not fix it
- they have a clear escalation path"

(or should that go in a different part of the doc?)